pkg/provider/azure_vmss.go

```diff
@@ -1253,7 +1253,7 @@ func (ss *ScaleSet) ensureVMSSInPool(service *v1.Service, nodes []*v1.Node, back
 			// in this scenario the vmSetName is an empty string and the name of vmss should be obtained from the provider IDs of nodes
 			resourceGroupName, vmssName, err := getVmssAndResourceGroupNameByVMProviderID(node.Spec.ProviderID)
 			if err != nil {
-				klog.V(4).Infof("ensureVMSSInPool: found VMAS node %s, will skip checking and continue", node.Name)
+				klog.V(4).Infof("ensureVMSSInPool: found VMSS node %s, will skip checking and continue", node.Name)
```

Comment:
This is not a typo. VMAS stands for virtual machine availability set.

Reply:
this PR would do the revert: https://github.com/kubernetes-sigs/cloud-provider-azure/pull/2981

Reply:
Never mind, I will revert it in my change.

Reply:
@nilo19 That is the correct logging for the VMSS function, it isn't trying to do anything with VMAS

Reply:
Ah, I see what you are saying @nilo19

Reply:
you are right